telemetry-attribute-utils/src/test/java/org/hypertrace/telemetry/attribute/utils/db/DbTelemetryAttributeUtilsTest.java

```diff
@@ -0,0 +1,305 @@
+
+    // no sql related attributes present
+    attributes = AttributeTestUtil.buildAttributes(
+        Map.of(
+            "span.kind",
+            AttributeTestUtil.buildAttributeValue("clinet")));
+    when(e.getAttributes()).thenReturn(attributes);
+    v = DbTelemetryAttributeUtils.getSqlURI(e);
+    assertTrue(v.isEmpty());
+  }
+
+  @Test
+  public void testGetBackendURIForOtelFormat() {
+    Event e = mock(Event.class);
+    // connection string is present
+    Attributes attributes = AttributeTestUtil.buildAttributes(
+        Map.of(
+            OTelDbAttributes.DB_CONNECTION_STRING.getValue(),
+            AttributeTestUtil.buildAttributeValue("mysql://127.0.0.1:3306")));
+    when(e.getAttributes()).thenReturn(attributes);
+    Optional<String> v = DbTelemetryAttributeUtils.getBackendURIForOtelFormat(e);
+    assertEquals("mysql://127.0.0.1:3306", v.get());
+
+    // only ip is present
+    attributes = AttributeTestUtil.buildAttributes(
+        Map.of(
+            OTelDbAttributes.NET_PEER_IP.getValue(),
+            AttributeTestUtil.buildAttributeValue("127.0.0.1")));
+    when(e.getAttributes()).thenReturn(attributes);
+    v = DbTelemetryAttributeUtils.getBackendURIForOtelFormat(e);
+    assertEquals("127.0.0.1", v.get());
+
+    // ip & host present
+    attributes = AttributeTestUtil.buildAttributes(
+        Map.of(
+            OTelDbAttributes.NET_PEER_IP.getValue(),
+            AttributeTestUtil.buildAttributeValue("127.0.0.1"),
+            OTelDbAttributes.NET_PEER_NAME.getValue(),
+            AttributeTestUtil.buildAttributeValue("mysql.example.com")));
+    when(e.getAttributes()).thenReturn(attributes);
+    v = DbTelemetryAttributeUtils.getBackendURIForOtelFormat(e);
+    assertEquals("mysql.example.com", v.get());
+
+    // host & port present
+    attributes = AttributeTestUtil.buildAttributes(
+        Map.of(
+            OTelDbAttributes.NET_PEER_IP.getValue(),
+            AttributeTestUtil.buildAttributeValue("127.0.0.1"),
+            OTelDbAttributes.NET_PEER_NAME.getValue(),
+            AttributeTestUtil.buildAttributeValue("mysql.example.com"),
+            OTelDbAttributes.NET_PEER_PORT.getValue(),
+            AttributeTestUtil.buildAttributeValue("3306")));
+    when(e.getAttributes()).thenReturn(attributes);
+    v = DbTelemetryAttributeUtils.getBackendURIForOtelFormat(e);
+    assertEquals("mysql.example.com:3306", v.get());
+  }
+
+  @Test
+  public void testGetDbTypeForOtelFormat() {
+    Event e = mock(Event.class);
+    Attributes attributes = AttributeTestUtil.buildAttributes(
+        Map.of(
+            OTelDbAttributes.DB_SYSTEM.getValue(),
+            AttributeTestUtil.buildAttributeValue(OTelDbAttributes.MONGODB_DB_SYSTEM_VALUE.getValue())));
+    when(e.getAttributes()).thenReturn(attributes);
+    Optional<String> v = DbTelemetryAttributeUtils.getDbTypeForOtelFormat(e);
+    assertEquals(OTelDbAttributes.MONGODB_DB_SYSTEM_VALUE.getValue(), v.get());
```

Comment:
Let's use literal constant string value for expected - "mongo"

Reply:
Isn't this better?